valkyrie/lib/valkyrie/persistence/fedora/metadata_adapter.rb

```diff
@@ -36,10 +36,7 @@ def connection_prefix
     end
 
     def wipe!
```

Comment:
I think we can just get rid of this, now that we have a correct spot to put it.

Reply:
👍